### PR TITLE
feat: support multiple VAST versions

### PR DESCRIPTION
**Pull request by oscnord**
This PR adds support for multiple VAST versions. Versions supported are 2, 3 and 4 where 4 is the default.

### CODE REVIEW

File: utils/vast-maker.js [hunk 1]
```diff
@@ -191,35 +216,39 @@ function AttachPodAds(vast4, podAds, params) {
           idRegistry: 'test-ad-id.eyevinn',
           idValue: encodeURIComponent(`${podAds[i].universalId}${i + 1}`),
         }
```
Comment (Nfrederiksen) [inline utils/vast-maker.js:218]: Whole PR is great. Nice attention to detail with the adId attribute. Weirdly enough IAB's own VAST validator claims that adID is not allowed but it clearly states in the VAST 3 specs that you canhave it. 

Anyway, ONE small change. The duplicated universalAdId must have been a typo from long ago. Do not remeber setting it on purpose, but there is no need for 2 universal ad ids in the same creative. 
I see that you perserved the extra `addUniversalAdId()` but you can go ahead and remove that

File: utils/vast-maker.js [hunk 2]
```diff
@@ -109,34 +111,46 @@ function VastBuilder(params) {
...
       .attachCreative({
         id: `CREATIVE-ID_00${i + podSize}`,
-        adId: `${ads[i].id}`,
+        [adId]: `${ads[i].id}`,
         sequence: `${i + podSize}`,
-      })
+      });
+    if (vast.attrs.version === "4.0") {
+      mediaNode = mediaNode
       .addUniversalAdId(encodeURIComponent(`${ads[i].universalId}${i + podSize}`), {
```
Comment (Nfrederiksen) [inline utils/vast-maker.js:154]: Remove this extra `addUniversalAdId()` as well

**Review by Nfrederiksen — CHANGES_REQUESTED** [review]

File: utils/vast-maker.js (hunk 1 above)
Comment (oscnord) [reply utils/vast-maker.js:218]: Which one of these is the correct one then?

https://github.com/Eyevinn/test-adserver/blob/b924921c658caf814a66af7b6c962f36da39b8a1/utils/vast-maker.js#L154-L164

The thing that differs is the `idRegistry`.

**Review by Nfrederiksen — APPROVED** [review]
LGTM sir

**Comment by oscnord** [comment]
@Nfrederiksen @birme Looks like I don't have access to merge the PR so feel free to do it whenever you feel like it.

**Pull request merged**